pkgs/top-level/all-packages.nix

```diff
@@ -3613,7 +3615,7 @@ with pkgs;
 
   mmv = callPackage ../tools/misc/mmv { };
 
-  morituri = callPackage ../applications/audio/morituri { };
+  whipper = callPackage ../applications/audio/whipper { };
```

Comment:
I think `whipper` should be placed further down the list to maintain the "pseudo-alphabetic" ordering of the file.